src/components/cards/cards.jsx

```diff
@@ -85,32 +85,46 @@ const CardHeader = ({onCloseCards, onShrinkExpandCards, onShowAll, totalSteps, s
     </div>
 );
 
-// Video step needs to know if the card is being dragged to cover the video
-// so that the mouseup is not swallowed by the iframe.
-const VideoStep = ({video, dragging}) => (
-    <div className={styles.stepVideo}>
-        {dragging ? (
-            <div className={styles.videoCover} />
-        ) : null}
-        <iframe
-            allowFullScreen
-            allowTransparency="true"
-            frameBorder="0"
-            height="257"
-            scrolling="no"
-            src={`https://fast.wistia.net/embed/iframe/${video}?seo=false&videoFoam=true`}
-            title="📹"
-            width="466"
-        />
-        <script
-            async
-            src="https://fast.wistia.net/assets/external/E-v1.js"
-        />
-    </div>
-);
+class VideoStep extends React.Component {
+    componentDidMount () {
+        const script = document.createElement('script');
+        script.src = `https://fast.wistia.com/embed/medias/${this.props.video}.jsonp`;
+        script.async = true;
+        document.body.appendChild(script);
+
+        const script2 = document.createElement('script');
+        script2.src = 'https://fast.wistia.com/assets/external/E-v1.js';
+        script2.async = true;
+        document.body.appendChild(script2);
+    }
+    componentDidUpdate () {
+        const video = window.Wistia.api(`${this.props.video}`);
+        if (!this.props.expanded) {
+            video.pause();
+        }
+    }
+    render () {
+        return (
+            // Video step needs to know if the card is being dragged to cover the video
+            // so that the mouseup is not swallowed by the iframe.
+            <div className={styles.stepVideo}>
+                {this.props.dragging ? (
+                    <div className={styles.videoCover} />
+                ) : null}
+                <div
+                    className={`wistia_embed wistia_async_${this.props.video}`}
+                    style={{height: `257px`, width: `466px`}}
+                >
+                    &nbsp;
+                </div>
+            </div>
+        );
+    }
+}
 
 VideoStep.propTypes = {
     dragging: PropTypes.bool.isRequired,
+    expanded: PropTypes.bool.isRequired,
     video: PropTypes.string.isRequired
```

Comment:
This idea is beyond the scope of this change, but I'd love to see the `video` prop renamed. What is it? A video Id? A video URL?

Reply:
I believe `video` is an id for Wistia embed URLs, like the value for `video` of `ir0j8ljsgm` is one I have stored for testing purposes (it's the id for the 'Make Music' tutorial video).